Sources/MapboxMaps/Viewport/ViewportStatusObserver.swift

```diff
@@ -0,0 +1,19 @@
+public protocol ViewportStatusObserver: AnyObject {
+    func viewportStatusDidChange(from fromStatus: ViewportStatus,
+                                 to toStatus: ViewportStatus,
+                                 reason: ViewportStatusChangeReason)
```

Comment:
this is a plain string on Android, we could consider make it a separate type to make it extendable in the future, will add to my todo list before RC.